src/Service/Api.php

```diff
@@ -1687,6 +1690,29 @@ public function supportsSizingApi(Project $project, EnvironmentDeployment $deplo
         return !empty($settings['sizing_api_enabled']);
     }
 
+    /**
+     * Checks if a project supports the Autoscaling API.
+     *
+     * @param Project $project
+     * @return bool
+     */
+    public function supportsAutoscaling(Project $project)
+    {
+        $cacheKey = 'project-capabilities:' . $project->id;
+        $cachedCapabilities = $this->cache->fetch($cacheKey);
+        if (!empty($cachedCapabilities) && !empty($cachedCapabilities['autoscaling'])) {
+            return (bool) $cachedCapabilities['autoscaling']['enabled'];
+        }
+
+        $capabilities = $this->getHttpClient()->get($project->getUri() . '/capabilities')->json();
+        $this->cache->save($cacheKey, $capabilities, $this->config->get('api.projects_ttl'));
```

Comment:
I think it would be neat to make this a `getProjectCapabilities($project)` method? and replace other calls to `$project->getCapabilities()`, which have their own kinds of cache right now.